Makefile

```diff
@@ -29,6 +33,8 @@ check-headers-file:
 
 production-build: build check-headers-file ## Build the production site and ensure that noindex headers aren't added
 
+linkcheck-build: build-linkcheck check-headers-file ## Build the production site and ensure that noindex headers aren't added
```

Comment:
Why is this target needed?

Reply:
@kbhawkey once again – in abundance of caution, this emulates the `production-build` step. I think we can skip it?

Reply:
I think skipping this would be fine.

Reply:
I would leave out any targets that are not needed to run the link checker.

Reply:
I'll remove this :+1: thanks for the feedback folks!